addons/telestation/fn_ui.sqf

```diff
@@ -0,0 +1,109 @@
+/* ----------------------------------------------------------------------------
+Function: cnto_telestation_fnc_telestation;
+Description:
+    Adds a telestation UI to an object. The telestation allows a player to teleport to any other player on their team, but will not teleport the player near or in the line of sight of enemies or friendlies.
+Parameters:
+    0: The item to attach the telestation to addaction to.
+Returns:
+    None
+Examples:
+    (begin example)
+        this call cnto_telestation_fnc_telestation;
+    (end)
+Author:
+    Seb
+---------------------------------------------------------------------------- */
+params ["_telestation"];
+
+private _fnc_UI = {
+    params ["_caller"];
+
+    [
+        "Telestation",
+        [
+            ["EDIT:MULTI", "The telestation is only to be used in cases of technical problems.", ["", {}, 0], true],
+            ["EDIT:MULTI", "Confirm your use is because of technical difficulties.", ["", {}, 0], true], 
```

Comment:
There is no label control?

Reply:
Nope! Lots of frustrating testing but this was the best way I found to do it.